db/keyval/etcdv3/README.md

```diff
@@ -28,10 +28,10 @@ You can compact Etcd using two ways.
 - using config file by setting `auto-compact` option to the duration of
   period that you want the Etcd to be compacted.
 
-## ETCD resynchronization
+## Reconnect resynchronization
 
 - If connection to the ETCD is interrupted, resync can be automatically called 
```

Comment:
to the ETCD or Consul (if you mean it in general since you've changed ETCD to Reconnect)

Reply:
Actually, both have separate READMEs and this one is for ETCD only. I just intended to make this section more clearly defined by mentioning the reason of resynchronization right in the title.